### ISSUE TITLE
Child theme on Multisite does not work

### ISSUE THREAD

**Issue by javiercasares** [WordPress/gutenberg#61608]
### Description

When activating a Child Theme in a Multisite it says:

_The parent theme is missing. Please install the "abc" parent theme._

Also, the child theme goes to the "Broken" tab.

_After doing the "step-by-step"_

When you go to the canva part, it broke `/wp-admin/site-editor.php?canvas=edit`

Related: #58549

### Step-by-step reproduction instructions

1. Install a new WP Multisite with 2 subsites (main + subsite)
2. Install the Create Block Theme plugin
3. On the main site, go to the Editor → Create Block Theme → go to the "new" canva editor (using the new options and not the old ones).
4. Go to the Create new theme from scratch.
5. Go to the Multisite options and activate the new theme.
6. Delete all the other themes.
7. Go, again, to the canva editor, and Create New theme → create a Child theme
8. Activate the Child Theme as Network (both parent and child may be activated)
9. Go to the subsite and activate the Child theme

### Environment info

- WordPress 6.5
- nginx
- PHP 8.2
- MariaDB 10.11
- No plugins

### Please confirm that you have searched existing issues in the repo.

Yes

### Please confirm that you have tested with all plugins deactivated except Gutenberg.

Yes

**Comment by javiercasares** [comment]
One interesting that also happened is, if I install the TT24 theme, and leave it activated on all the sites / subsites and deactivate the parent theme, then, the Child Theme looks like it works because goes from the Broken Tab to the All tab.

**Comment by javiercasares** [comment]
I think I found it… it needs an "index.php". If there is no file there, it doesn't work.

**Comment by nicofyve** [comment]
Adding an empty index.php to TT24 on my multisite installation solved the problem for me, too. Thank you @javiercasares !

EDIT: By chance I found another way to work around this issue:

When I encountered the issue mentioned in the OP, I had the plugin _Create Theme Block_ Network Activated.

The OP issue also disappears for me when the plugin _Create Theme Block_ is NOT Network Activated (without the index.php file added to the TT24 theme). It seems fine to have the plugin activated for the individual site.

**Comment by luminuu** [comment]
Can confirm this is an issue with a missing index.php in a block theme that is used as a parent theme in a multisite environment.

**Comment by carolinan** [comment]
I am not able to reproduce this using the following steps:

- WordPress: 6.7-alpha-58576-src
- PHP: 7.4.30
- Server: nginx/1.23.1
- Database: mysqli (Server: 8.0.36 / Client: mysqlnd 7.4.30)
- Browser: Chrome 126.0.0.0
- OS: macOS
- Theme: (various themes tested)
- MU Plugins: None activated
- Plugins:
  * Create Block Theme 2.3.0
  * Health Check & Troubleshooting 1.7.0
  * Test Reports 1.1.0
  * With and without Gutenberg 18.7.0.
 
Enable multisite that uses the path, not the subdomain.
Network enable the parent theme (twenty twenty-four) and two child themes: one classic, one block.
✅  No issues with either theme on activation.

With the classic theme active, the Site Editor does not load the PHP template from my child theme. The preview looks like it is trying to load something, and there is no message saying that the current template is not editable. Is this what you mean with "When you go to the canva part, it broke /wp-admin/site-editor.php?canvas=edit"  @javiercasares?

----
Using Create Block Theme Version 2.3.0 to create:

1) a new parent block theme directly on the subsite, or
2) a new child block theme directly on the subsite,

redirects me to /wp-admin/site-editor.php?canvas=edit which does not load.
It seems more like a plugin problem to me, where it should not be redirecting to canvas=edit ?

**Comment by carolinan** [comment]
Using Create Block Theme Version 2.3.0 to create:
1) a new parent block theme directly on the subsite, or
2) a new child block theme directly on the subsite,
redirects me to /wp-admin/site-editor.php?canvas=edit which does not load. 
If I return to the WordPress admin, the themes is activated correctly and I can access the Site Editor and front from there, where the theme is displaying correctly.

It seems more like a plugin problem to me, where it should not be redirecting to canvas=edit ?

**Comment by carolinan** [comment]
I am testing the same steps on Windows 11:

- WordPress: 6.7-alpha-58576-src
- PHP: 8.2.12
- Server: nginx/1.25.3
- Database: mysqli (Server: 8.0.37 / Client: mysqlnd 8.2.12)
- Browser: Chrome 126.0.0.0
- OS: Windows 10/11
- Theme: (various themes tested)
- MU Plugins: None activated
- Plugins:
  * Create Block Theme 2.3.0
  * Test Reports 1.1.0
  * With and without Gutenberg 18.7.1.

### Manually created child themes:
Both the classic child theme and the block child theme activate without problems or error messages.

### Create Block Theme:

- When I am **on the subsite** and I use Create Block Theme to create a new parent theme, the parent theme is not immediately activated, nor network enabled.
- When I am on the subsite and I use Create Block Theme to create a new child theme of Twenty Twenty-Four, the child theme is not immediately activated, nor network enabled.

After creating the theme and pressing OK to reload the browser, it looks like the theme is active, but its not since it is not network enabled. Refreshing the theme screen in the admin shows that the new theme is not active.
Once network enabled, the child theme can be activated without error messages.

**Comment by carolinan** [comment]
I was able to test sub domains using "Local", WordPress 6.5.5 and 6.6 RC2 on macOS and Windows11, both installations using PHP 8.1.29, and I can not reproduce the issue where the parent theme is missing.

**Comment by luminuu** [comment]
Have you also checked you had the index.php removed from the parent block theme?

**Comment by carolinan** [comment]
Yes, neither Twenty Twenty-Four or the parent theme created with create block theme has an index.php.

**Comment by carolinan** [comment]
My classic child theme that I manually created, did not include theme.json.

**Comment by luminuu** [comment]
I have been trying to reproduce this but can't do it either.

**Comment by carolinan** [comment]
There are still multiple issues open for this on Trac though.
Perhaps, as suggested in https://core.trac.wordpress.org/ticket/57991, the parent theme was not network enabled?

**Comment by luminuu** [comment]
Nope, disabling the parent theme network wide does not have any effect on the subsite with the active child theme. It still works for me.

**Comment by SirLouen** [comment]
It seems that `Needs Testing` is not applicable any more. 

Furthermore, I'm not convinced if this report belongs to Core at this point given the reported actions

For now, time to remove `Needs Testing` and closing this for [reported- upstream](https://core.trac.wordpress.org/ticket/57991)

cc @jordesign @luminuu @carolinan

**Comment by SirLouen** [comment]
## Reproduction Report
### Description
Adding on top of my last comment, I wanted to see if this was still reproducible in 2025.

❌ This report can't validate that the issue can be reproduced.

### Environment
- WordPress: 6.9-alpha-60459
- PHP: 8.2.29
- Server: nginx/1.29.0
- Database: mysqli (Server: 8.4.5 / Client: mysqlnd 8.2.29)
- Browser: Chrome 138.0.0.0
- OS: Windows 10/11
- Theme: Child Theme
- MU Plugins: None activated
- Plugins:
  * Create Block Theme 2.7.0
  * Test Reports 1.2.0

### Reproduction Instructions
1. Followed the exact same steps than provided in the OP (installing Create Block Theme plugin also)
2. ❌ There is no error in the child theme (attaching a Screencast)

### Actual Results
1. ❌Error condition is not occurring.

### Additional Notes
- Maybe, given that this problem happened a while ago and as @carolinan has already pointed out, multiple reports have arose in Core regarding this topic, maybe it was solved at some point. Anyway if @javiercasares happen to find something, or if its easily reproducible in WP Playground it would be greater.
- For now I think this report could be closed as `worksforme`

cc @Mamaduka 

### Supplemental Artifacts

Screencast: https://f003.backblazeb2.com/file/wordpress-videos/wp-videos/2025/07/gb61608.mp4
